ci/checks/copyright.py

```diff
@@ -189,8 +188,15 @@ def checkCopyright_main():
                            help="If set, "
                            "only files seen as modified by git will be "
                            "processed.")
+    argparser.add_argument("--exclude",
+                           dest='exclude',
+                           action="append",
+                           required=False,
+                           default=["python/cuml/_thirdparty/"],
+                           help="Exclude the paths specified (regexp).")
 
     (args, dirs) = argparser.parse_known_args()
+    ExemptFiles = [re.compile(pathName) for pathName in args.exclude]
```

Comment:
Can this fail on an invalid RegEx? If so, I would add a simple try/except block here just in case.

Reply:
I added the try/except in case of a compilation error of the RegEx